app/qml/ProjectPanel.qml

```diff
@@ -527,6 +543,318 @@ Item {
+
+              anchors.centerIn: avatarImage
+              source: InputStyle.accountIcon
+              height: avatarImage.height * 0.8
+              width: height
+              sourceSize.width: width
+              sourceSize.height: height
+              fillMode: Image.PreserveAspectFit
+            }
+
+            ColorOverlay {
+              anchors.fill: userIcon
+              source: userIcon
+              color: "#FFFFFF"
+            }
+          }
+        }
+      }
+
+      background: Rectangle {
+        anchors.fill: parent
+        color: InputStyle.clrPanelMain
+      }
+
+      Item {
+        id: pageContent
+
+        anchors.fill: parent
+
+        states: [
+          State {
+            name: "local"
+          },
+          State {
+            name: "created"
+          }
+        ]
+
+        onStateChanged: {
+          __merginApi.pingMergin()
+          refreshProjectList()
+          pageFooter.setActiveButton( pageContent.state )
+        }
+
+        Connections {
+          target: root
+          function onVisibleChanged() {
+            if ( root.visible ) { // projectsPanel opened
+              pageContent.state = "local"
+            }
+            else {
+              pageContent.state = ""
+            }
+          }
+
+          function onResetView() {
+            if ( pageContent.state === "created" )
+              pageContent.state = "local"
+          }
+        }
+
+        ComboBox {
+          id: workspaceComboBox
+
+          model: __merginApi.userInfo.workspaces
+          onActivated: function(index) {
+            console.log("Switch to", index)
+          }
+        }
+
+        StackLayout {
+          id: projectListLayout
+
+          anchors.fill: parent
+          currentIndex: pageFooter.currentIndex
+
+          ProjectListPage {
+            id: localProjectsPage
+
+            projectModelType: ProjectsModel.LocalProjectsModel
+            activeProjectId: root.activeProjectId
+            list.visible: !stackView.pending
+
+            onOpenProjectRequested: function( projectId, projectFilePath ) {
+              setupProjectOpen( projectId, projectFilePath )
+            }
+            onShowLocalChangesRequested: function( projectId ) {
+              showChanges( projectId )
+            }
+            list.onActiveProjectDeleted: setupProjectOpen( "", "" )
+          }
+
+          ProjectListPage {
+            id: createdProjectsPage
+
+            projectModelType: ProjectsModel.CreatedProjectsModel
+            activeProjectId: root.activeProjectId
+            list.visible: !stackView.pending
+
+            onOpenProjectRequested: function( projectId, projectFilePath ) {
+              setupProjectOpen( projectId, projectFilePath )
+            }
+            onShowLocalChangesRequested: function( projectId ) {
+              showChanges( projectId )
+            }
+            list.onActiveProjectDeleted: setupProjectOpen( "", "" )
+          }
+        }
```

Comment:
In the ticket we have this

> Shared with me and explore will be empty since they do not have counterparts in workspace approach.

But added per your request.